### PR TITLE
Add JPetMCHit 

### PR DESCRIPTION
**Pull request by wkrzemien**
1)JPetMCHit to be used to load info from GATE/GEANT
2) Also, remove pqxx from travis.yml - it is not needed anymore

### CODE REVIEW

File: MC/JPetMCHit/JPetMCHit.h [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  void setMCVtxIndex(Int_t vi);
+  void setPolarization(Int_t xx, Int_t yy, Int_t zz);
+  void setMomentum(Float_t dd, Float_t ee, Float_t ff);
+
+private:
+
+  Int_t fMCDecayTreeIndex = 0;
+  Int_t fMCVtxIndex = 0;
+  TVector3 fPolarization; 	// initial polarisation of gamma photon
+  TVector3 fMomentum;     	// initial momentum of gamma photon
```
Comment (alekgajos) [inline MC/JPetMCHit/JPetMCHit.h]: Isn't this information redundant with respect to what will be available in the decay tree object?
Or do we foresee a usage of JPetMCHit without corresponding decay trees?

File: MC/JPetMCHit/JPetMCHit.h [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+  void setMCDecayTreeIndex(Int_t dti);
+  void setMCVtxIndex(Int_t vi);
+  void setPolarization(Int_t xx, Int_t yy, Int_t zz);
+  void setMomentum(Float_t dd, Float_t ee, Float_t ff);
+
+private:
+
+  Int_t fMCDecayTreeIndex = 0;
+  Int_t fMCVtxIndex = 0;
```
Comment (alekgajos) [inline MC/JPetMCHit/JPetMCHit.h]: These indices could in principle be unsigned.

Comment (wkrzemien) [reply MC/JPetMCHit/JPetMCHit.h]: ok. I will change it.

Comment (wkrzemien) [reply MC/JPetMCHit/JPetMCHit.h]: Done.

**Pull request merged**